### PR TITLE
EAG import fine tuning and password reset flow cleanup

### PR DESCRIPTION
**Pull request by viktor-yunenko**
Hi @sebgrebe, I prepared few updates, especially with the password reset flow.

We need to deploy this before the imports.

You can check them here:
- https://eahub-eagimport-b73ae45.us.aldryn.io/accounts/password/reset/
- https://eahub-eagimport-b73ae45.us.aldryn.io/accounts/password/reset/done/
- https://eahub-eagimport-b73ae45.us.aldryn.io/accounts/password/reset/key/mas-5pp-4a675f08a867cb2a5659/

### CODE REVIEW

**Comment by viktor-yunenko** [comment]
Although I can clean up few strings before the review.

**Comment by sebgrebe** [comment]
image

Minor/optional: Can we can "eahub.org" in the signature to "EA Hub" or "The EA Hub Admins" or maybe even one of our names, e.g., Vaidehi's?

**Comment by sebgrebe** [comment]
Password reset flow looks fine

File: eahub/templates/account/password_reset_from_key.html [hunk 1]
```diff
@@ -1,16 +1,62 @@
...
+  
+    <form method="POST">
+      {% csrf_token %}
+    
+      {% if token_fail %}
+        <div class="form-group mb-4">
+          <p class="error">
+            Unfortunately this password reset url isn't valid anymore.
+            Please try to <a href="{% url "account_reset_password" %}">reset your password here</a>,
+            or reach out to the support at contact@eahub.org.
```
Comment (sebgrebe) [inline eahub/templates/account/password_reset_from_key.html:25]: Maybe rather tech@eahub.org.?

Also, we should set this in environment variables or with django singleton (don't need to do this now)

**Review by sebgrebe — APPROVED** [review]

File: eahub/templates/account/password_reset_from_key.html (hunk 1 above)
Comment (viktor-yunenko) [reply eahub/templates/account/password_reset_from_key.html:25]: I would recommend reserving tech@ for technical matters only, eg sentry or DNS monitoring.

Ideally I would put there support@eahub.org, but looks like it wasn't set up.

Would that make sense to you?

**Comment by viktor-yunenko** [comment]
> Minor/optional: Can we can "eahub.org" in the signature to "EA Hub" or "The EA Hub Admins" or maybe even one of our names, e.g., Vaidehi's?

I would like to. This isn't our email though, it comes with the allauth package. I guess it's a lover priority comparing to other things as SSO and stuff? I'll create a ticket though.

**Pull request merged**